src/components/PokemonText.jsx

```diff
@@ -1,10 +1,11 @@
 import { usePokemon } from "../hooks/usePokemon";
+import { usePokemonBatch } from "../hooks/usePokemonBatch";
 import { capitalize } from "../util/capitalize"
 import "./PokemonText.css";
 
 export function PokemonText({ name, displayName = undefined }) {
-  const entry = usePokemon(name);
-
+  const entries = usePokemonBatch(name ? [name.toLowerCase()] : []);
+  const entry = entries && entries.length > 0 ? entries[0] : null;
```

Comment:
This can be simplified
```suggestion
  const entry = entries?.length > 0 ? entries[0] : null;
```